src/static/css/cf-theme-overrides.less

```diff
@@ -35,7 +35,7 @@
 @link-underline:                @pacific;
 @link-text-visited:             @teal;
 @link-underline-visited:        @teal;
-@link-text-hover:               @pacific-50;
```

Comment:
Good catch. So I set this because, the links in the `.category-slug` class were @pacific-80 [here](https://github.com/cfpb/cfgov-refresh/pull/868/files#diff-12248e16f0f10cdbbae81b9d3f021348L364), which CF sets using [`@category-slug-hover`](https://github.com/cfpb/cf-typography/blob/gh-pages/src/cf-typography.less#L271), which we set to [`@link-text-hover`](https://github.com/cfpb/cfgov-refresh/blob/flapjack/src/static/css/cf-theme-overrides.less#L226). So either `@category-slug-hover` in `cf-theme-overrides.less` should be set to `@pacific-80` or this should be reverted, in which case this PR lightens the links in category slugs.

I think this should be reverted. I don't see why the category slug links should be darker. For this in action see /newsroom/ and hover over the a category slug like `Speech` vs the links at the top of the content area.